mne/io/egi/tests/test_egi.py

```diff
@@ -115,6 +115,7 @@ def test_io_egi_mff():
                            test_scaling=False,  # XXX probably some bug
                            )
     assert raw.info['sfreq'] == 1000.
+    assert len(raw.info['dig']) == 132  # 128 eeg + 1 ref + 3 cardinal points
```

Comment:
`ref` should be the first of the digitized EEG points, and its position should show up in `info['chs'][ii]['loc'][3:6]` (first three elements are electrode loc, next three are ref loc). Can you implement and ensure those conventions are enforced here?